parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -247,6 +255,7 @@ try
     . c:\AzureData\windows\containerdfunc.ps1
     . c:\AzureData\windows\kubeletfunc.ps1
     . c:\AzureData\windows\kubernetesfunc.ps1
+    . c:\AzureData\windows\nvidiagpudriverfunc.ps1

```

Comment:
kuberneteswindowssetup.ps1 is released through Agentbaker svc by Node SIG,  nvidiagpudriverfunc.ps1 will be included in a ZIP file released through other pipeline by Windows SIG. If the AgentbakerSvc containing this change is released earlier than the release of Windows CSE zip file containing nvidiagpudriverfunc.ps1. The Windows non-GPU nodepool will fail because non-existing ps1 is referenced.

To be safe, can we change line 472 to something like
 $RebootNeeded = [System.Convert]::ToBoolean("true");
if ($global:ConfigGPUDriverIfNeeded) {
    . c:\AzureData\windows\nvidiagpudriverfunc.ps1
   $RebootNeeded = Start-InstallGPUDriver -EnableInstall $global:ConfigGPUDriverIfNeeded -DriverUrlConfig $DriverUrlConfig
}

Reply:
This is a draft PR. The update should be separated into below parts.
1. Update AKS Windows CSE scripts
2. Release a new AKS Windows CSE script package
3. Update parts/windows/kuberneteswindowssetup.ps1 with new AKS Windows CSE script package version